alf/utils/summary_utils.py

```diff
@@ -16,6 +16,7 @@
 import tensorflow as tf
 from tensorboard.plugins.histogram import metadata
 from tensorflow.python.ops import summary_ops_v2
+from alf.utils.conditional_ops import run_if
```

Comment:
shall we import from common.py for consistency? not sure what's the best practice here

Reply:
Perhaps we should import every thing under alf.util from alf.utils in the future so that you need not concern which module a symbol is from. But even with that, for modules under util, you have to import from the actual source of a symbol to prevent circular import.